odoo/addons/base/models/res_company.py

```diff
@@ -63,7 +60,7 @@ def _get_default_favicon(self, original=False):
     partner_id = fields.Many2one('res.partner', string='Partner', required=True)
     report_header = fields.Text(string='Company Tagline', help="Appears by default on the top right corner of your printed documents (report header).")
     report_footer = fields.Text(string='Report Footer', translate=True, help="Footer text displayed at the bottom of all reports.")
-    logo = fields.Binary(related='partner_id.image_1920', default=_get_logo, string="Company Logo", readonly=False)
+    logo = fields.Binary(related='partner_id.image_1920', string="Company Logo", readonly=False)
```

Comment:
So the default company logo is changed to use a default in views instead of in the model? Are you sure you updated everywhere a company logo was used to add placeholder?

Reply:
We don't need a default logo anymore since it uses the initials avatar.

Reply:
This is a related to image not to avatar, it doesn't have initials like this.